File: Rust/SquareSum/MySolution.rs
fn square_sum(vec: Vec<i32>) -> i32 {
    let mut result = 0;
    for number in &vec {
        result += number*number;
    }    
    return result;
}

// Original kata: https://www.codewars.com/kata/515e271a311df0350d00000f
// My solution: https://www.codewars.com/kata/reviews/5824a8aa304383a19900005a/groups/63e9107426aff20001ebbc9b

File: PHP/Multiples3Or5/MySolution.php
<?php
  
function solution(int $number): int {
  $result = 0;
  for ($i=3; $i < $number ; $i++) { 
      if ($i % 3 == 0 || $i % 5 == 0) {
          $result += $i;
      }
  }
  return $result;
}

// Original kata: https://www.codewars.com/kata/514b92a657cdc65150000006
// My solution: https://www.codewars.com/kata/reviews/578840b8ea6a1ce290000002/groups/67b44061f2a2ca83dfbade11
